troubleshooting/missing-image-asset-causes-ios-publish-to-fail.md

```diff
@@ -0,0 +1,41 @@
+---
+title: Error Publishing App to the iOS AppStore
+description: Publishing Application to the AppStore is Failing due to Missing Image Assets
```

Comment:
Publishing to the Apple App Store fails with an ITMS-90032: Invalid Image Path error.